build/ttGen.ps1

```diff
@@ -0,0 +1,10 @@
+Write-Host "Regenerating all generated files from all .tt files."
```

Comment:
move this to `scripts\`

Reply:
When will this be run? Manually?

Reply:
moved to \scripts\utils and added some instructions.